func_tests/test_client.py

```diff
@@ -36,3 +38,28 @@ def test_record_sets(vinyldns_test_context):
     assert change.record_set.name == rs.name
     assert all([l.__dict__ == r.__dict__ for l, r in zip(change.record_set.records, rs.records)])
 
+
+def test_batch_change(vinyldns_test_context):
+    changes = [
+        AddRecord('change-test.vinyldns.', RecordType.A, 200, AData('192.0.2.111')),
+        AddRecord('192.0.2.111', RecordType.PTR, 200, PTRData('change-test.vinyldns.'))
+    ]
+    r = vinyldns_test_context.client.create_batch_change(
+        BatchChangeRequest(changes, 'comments', vinyldns_test_context.group.id)
+    )
+    assert r.user_id == 'ok'
+    assert r.user_name == 'ok'
+    assert r.comments == 'comments'
+
+    change1 = r.changes[0]
+    assert change1.input_name == 'change-test.vinyldns.'
```

Comment:
Might be worthwhile to create a `test_single_change` to do the `assert`s similar to the helpers above.

Reply:
For this test purposes, I like having these called out here.